not-invented-here.md

```diff
@@ -0,0 +1,108 @@
+* Fear of losing control. 
+* Security can be a problem.
+
+* Teams tightly control their processes/engagement models
+* Contributing teams do not read contrib.md or prepare ("fling it over the wall")
+* Conflicting business goals/individual performance measures may result in groups or individuals being dis-incented to engage
+* Fear of diluting your perceived value
+* Teams are more comfortable being held accountable for their own work - hesitant to take on unknown risk- ex: will i break my own product if I accept this? will I break stuff on a wider scale?
+* Teams are hesitant to reuse/consume others' code
+* Perceived value of accepting contributions that have narrow impact
+
+- Team egos
+- Individual egos 
+- The obvious benefit of what the solution could bring
+- Optics challenges - teams being judged for bringing in software from outside the walls
+- What metrics are being measured
+- Reward system 
+
+
+## Sketch (optional)
+visual illustration
+
+## Solutions
+*Verified resolutions and possible resolutions to the problem*
+* [Blog article from Oana-Maria Pop on going deep into the not-invented-here syndrome](http://blog.hypeinnovation.com/the-not-invented-here-syndrome) touches on this topic, describing a prescription for overcoming the "Not Invented Here" mindset: 
+    - "Acknowledge that the "Not Invented Here" mindset (NIH) exists
+    - Assess the impact of NIH on your innovation efforts. For example, have you missed opportunities?
+    - Build in explicit incentives to overcome NIH
+    - engage people outside of the organization in strategy/evaluation phases for fresh perspectives"
+* DMS video [Open Innovation: Proudly Found Elsewhere](https://www.youtube.com/watch?v=jNNz9poyKJs): Discusses the shift from strict NIH to empowering the "Proudly Found Elsewhere" approach. "It pays to look outside ones area (open innovation). The open Innovation funnel has permeable walls-->greater chance of success, greater speed."
+
+Provide template to use for code submission requirements. (Must include testing. Many companies have built-in, automated testing. Documented communication) For both host and contributors. Requires/assumes compliance.
+Mentorship (requirements can be informed via tracking above results)
+Talking is good /relationship building
+Finding incentives to drive IS behavior (incentives and measures can vary at a team level)
+Shift to a "profoundly found elsewhere" culture
+Demonstrate organizational interest in outside opinions
+Identify influencers who agree to be early adopters - set the stage for others
+Contributability is a mark of quality
+
+
+## Desired Resulting Context
+What is the situation after the problem has been solved?
+Developers search for and leverage existing options as opposed to rewriting. This results in: increased efficiency, increased reuse, higher levels of developer satisfaction, increased speed to market.
```

Comment:
Developers interact with code and products produced by others with the same trust and engagement as those that they themselves have produced.

Reply:
I have worked this feedback. Thanks!